backend/src/core/routes/tests/campaign.routes.test.ts

```diff
@@ -74,6 +75,168 @@ describe('GET /campaigns', () => {
       ]),
     })
   })
+
+  test('List campaigns order by created at', async () => {
+    for (let i = 1; i <= 3; i++) {
```

Comment:
haha I think it's more idiomatic to use functional instead of imperative programming with TypeScript (i.e. to use `map` instead of for loops here)

(totally a nit / opinionated take, feel free to ignore)